clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -95,6 +95,18 @@ void SYCL::constructLLVMForeachCommand(Compilation &C, const JobAction &JA,
                                          Foreach, ForeachArgs, None));
 }
 
+static llvm::SmallVector<StringRef, 10> SYCLDeviceLibList{
+    "crt",
+    "cmath",
+    "cmath-fp64",
+    "complex",
+    "complex-fp64",
+    "fallback-cassert",
+    "fallback-cmath",
+    "fallback-cmath-fp64",
+    "fallback-complex",
+    "fallback-complex-fp64"};
```

Comment:
Does this mean that if we were to add a new device library, we'd have to put its base name here as well? Or is it that no new library is likely to be added?

Reply:
Currently, those device libraries can meet our requirements but we may add new ones in the future.

Reply:
The explanatory comment you've added should help, I think. I wonder if, in the future, we could make some sort of a configuration file with the device library base names, parsing that both in the relevant CMakeLists and in the driver code (via `#include` for compile-time?). I understand this is somewhat a vague approach - for example, where should we put the configuration file so that it could be conveniently accessed from all relevant LLVM subprojects? But just as something to consider for future improvements.